src/ringmesh/mesh/mesh.cpp

```diff
@@ -1198,14 +1183,14 @@ namespace RINGMesh
         volume_mesh = VolumeMesh3D::create_mesh( type );
     }
 
-    template class mesh_api MeshBase< 2 >;
+   // template class mesh_api MeshBase< 2 >;

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this block of commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3ACommentedCode)